app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/CustomizeHomeButtonViewHolder.kt

```diff
@@ -48,6 +48,8 @@ class CustomizeHomeButtonViewHolder(
                     CustomizeHomeButton(
                         onButtonClick = { interactor.openCustomizeHomePage() }
                     )
+
+                    Spacer(modifier = Modifier.height(88.dp))
```

Comment:
Just wanted to state that we are not guaranteed to always render this. https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlView.kt#92

Reply:
Although looking at the issue, this should be fine, but might be worthwhile to test where the "Customize home" button is not present.

Reply:
I'd considered using a standalone view for just the spacing. That feels the most foolproof to me, but it felt like a bit of an awkward solution. I figured the likelihood of the `CustomizeHomeButtonViewHolder` not being shown was low, given the amount conditions that triggered it.

I was also relying on the Figma designs available for the home page. They show the button as always visible, so I thought this would probably be good enough. Are there conditional layouts in Figma that I'm missing? New to the tool.

Happy to add a standalone view for the spacing. It might make the code more explicit anyway, rather than hiding in whatever happens to be the last view.

Reply:
Ya... it's an awkward situation. Without testing this myself, my current guesses is that a standalone view is probably most foolproof as you stated. If I was in your shoes, I would only gain more confidence of the current solution by seeing how it looks without the "Customize home" button, also consider that the various views like "Recent Bookmarks", "Collections", 'Jump back in", "Pocket" and "Recent History" are all customizable and conditional. I am wondering what happens to the bottom spacing for all the different combination of views being visible and not visible. I guess what we are looking for here is just a spacing below the last visible view and the toolbar.

Reply:
Alternatively, I would look at refactoring the various views to add a bottom spacer to each view so we aren't adding the spacer at the top of the view only when they are render. See the 68dp spacer in this view as an example.

Reply:
Of course, after thinking about all these troubling possibilities. It is possible that simple is best here and adding the standalone view is the way to go, but I leave that to your investigation.

Reply:
The bottom still gets cut off when the `customize homepage` button isn't present, so good call there.

I'll go ahead and add a standalone spacer item for now. It'd be much easier to add bottom spacing to all the views if compose was being used throughout the home screen, but I'm hesitant to add/update custom spacing to each individual item as is.

Reply:
Updated!